Containerfile

```diff
@@ -41,7 +41,15 @@ RUN wget https://github.com/docker/compose/releases/latest/download/docker-compo
     install -c -m 0755 /tmp/docker-compose /usr/bin
 RUN systemctl enable docker.socket && \
     systemctl enable podman.socket
+
+RUN rpm-ostree install -y java-17-openjdk && \
```

Comment:
could you move the installation of java to the top of the Containerfile, with the other ones?
https://github.com/ii/image/pull/2/files#diff-5fcdf9b4580789697d834d1456a22bcfaa236d668fc180cad4775afc36ed5914R12